### DISCUSSION TITLE
about connecting with Flightgear

### DISCUSSION THREAD

**Discussion by dyoung666** [JSBSim-Team/jsbsim#733] [Q&A]
I used jsbsim and flightgear for the first time ,and I notice that there are the output script“data_output/flightgear.xml”.
I get jsbsim by > conda install jsbsim directly, so how to use this script?
if i only input "jsbsim -realtime data_output/flightgear.xml scripts/c1723.xml", it can't work.
I'm inexperienced. I'm sorry to ask such a question

**Comment by bcoconni** [comment]
Hi @dyoung666, the process to interface JSBSim with FlightGear is described in our FAQ [How can I interface JSBSim with FlightGear ?](https://github.com/JSBSim-Team/jsbsim/wiki/Frequently-Asked-Questions#how-can-i-interface-jsbsim-with-flightgear-)

**Reply by bcoconni** [reply]
You need to provide the relative path to the files `scripts/Short_S23_2.xml` and `data_output/flightgear.xml` to JSBSim via the `--root` option.

Say the files are located in `/home/dyoung666/JSBSim/data_files` then you shall run the following command:
```bash
> JSBSim --root=/home/dyoung666/JSBSim/data_files scripts/Short_S23_2.xml data_output/flightgear.xml --realtime --nice
```

**Comment by dyoung666** [comment]
Thank you for your reply！ I think it works！
